.github/workflows/ci-workflow.yml

```diff
@@ -326,6 +326,44 @@ jobs:
       with:
         name: premake-solaris-${{ matrix.platform }}
         path: bin/${{ matrix.config }}/
+  omnios:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        config: [debug, release]
+        platform: [x64]
+        cc: [gcc]
+    timeout-minutes: 30
+    defaults:
+      run:
+        shell: omnios {0}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Start OmniOS VM
+      uses: vmactions/omnios-vm@v1
+      with:
+        usesh: true
+        prepare: |
+          pkg install build-essential web/ca-bundle
+    - name: Build
+      run: |
+        cd $GITHUB_WORKSPACE
+        CC=${{ matrix.cc }} PLATFORM=${{ matrix.platform }} CONFIG=${{ matrix.config }} PREMAKE_OPTS="--cc=${{ matrix.cc }}" ./Bootstrap.sh
```

Comment:
Are both `CC=${{ matrix.cc }}` and `"--cc=${{ matrix.cc }}"` required?
Some jobs have both, some don't.

Reply:
This is copy-pasted, but yes it is required sometimes.

`CC` is for `Bootstrap.mak`. Some of these systems don't have `cc` set to anything, so we need to specify via `CC`.
`--cc` is for Premake. Really only required when using Clang, but not really worth removing.